resources_pages/mds-help.sh

```diff
@@ -0,0 +1,180 @@
+# Detect which file browser string to use
+if [[ "$(uname)" == 'Linux' ]]; then
+    browser_cmd='xdg-open .'
+elif [[ "$(uname)" == 'Darwin' ]]; then
+    browser_cmd='open .    '
+elif [[ "$OSTYPE" == 'msys' ]]; then
+    browser_cmd='explorer .'
+else
+    echo "Operating system verison could not be detected." >> check-setup-mds.log
+fi
+
+# Use colors for headings for clarity
+ORANGE='\033[0;33m'
+NC='\033[0m' # No Color
+
+# Help messages
+help_preamble="
+This help message shows a brief explanation
+of the most important terminal commands we use in MDS.
+It is meant to be complement the lecture notes
+and be used as a reference when you forget a command.
+Note that the words 'directory' and 'folder' are used interchangeably.
+This also includes the aliases we have created in your bash configuration files
+for commands that are very useful and that we want encourage you to use often.
+
+All the commands are printed by default,
+but you can specifiy a single section to show
+by using the subcommands 'shell', 'git', 'conda', or 'alias'.
+For example, if you only want to see the shell command help you would type:
+
+mds-help shell
+"
+
+shell_help="
+${ORANGE}# Shell commands${NC}
+
+These are typed into your bash terminal prompt.
+
+~                               Nickname for the home folder ($HOME)
+.                               Nickname for the current folder
+..                              Nickname for the parent folder
+$browser_cmd                      Open the current folder in your file browser
+
+man <COMMAND>                   Show the manual (help) of a command
+history                         List all commands you have typed
+pwd                             Print the name of the current (working) directory
+
+ls                              List the content of the current directory
+ls <FOLDER>                     List the content of a specific directory
+ls -a                           List hidden files
+ls -l                           List additional information for each file
+
```

Comment:
Just a tiny suggestion to include `ls -lh` as well for making the files sizes human-readable.

Reply:
I have this in one of the aliases but will add here as well